test/e2e/tests/chaos.go

```diff
@@ -214,6 +215,19 @@ func (t *StorageMigratorChaosTest) Test(done <-chan struct{}) {
 			if len(ss.Status.PersistedStorageVersionHashes) == 1 && ss.Status.PersistedStorageVersionHashes[0] == ss.Status.CurrentStorageVersionHash {
 				continue
 			}
+			listOfMigrations, err := t.migrationClient.MigrationV1alpha1().StorageVersionMigrations().List(metav1.ListOptions{})
+			if err != nil {
+				util.Logf("unable to get list of storage version migrations: %s", err)
+			} else {
+				for _, migration := range listOfMigrations.Items {
```

Comment:
nesting this an extra level deep looks questionable to me.  If you want to have this information, why don't you dump it somewhere where you aren't already in a loop of other resources?